src/server/game/Entities/Player/CollectionMgr.h

```diff
@@ -78,12 +80,25 @@ class TC_GAME_API CollectionMgr
     HeirloomContainer const& GetAccountHeirlooms() const { return _heirlooms; }
 
     // Account-wide mounts
+    void LoadMounts();
+    void LoadAccountMounts(PreparedQueryResult result);
+    void SaveAccountMounts(SQLTransaction& trans);
+    void MountSetFavorite(uint32 spellId, bool favorite);
+
+    bool UpdateAccountMounts(uint32 spellId, bool isFavourite /*= false*/);
+    bool AddMount(uint32 spellId, bool isFavourite /*= false*/, bool factionMount = false);
+    static void LoadMountDefinitions();
+ 
+    MountContainer const& GetAccountMounts() const { return _mounts; }
 
 private:
     WorldSession* _owner;
 
     ToyBoxContainer _toys;
     HeirloomContainer _heirlooms;
+    MountContainer _mounts;
+
+    static MountDefinitionMap _mountDefinitions;
```

Comment:
I did that stuff some time ago, before you linking changes, so didn't look on it from this angle, but thanks, will fix it